MAUI/DataGrid/Getting-started.md

```diff
@@ -0,0 +1,336 @@
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+            ColumnSizer="Star"
+            AutoGenerateColumnsMode="None"
+            ItemsSource="{Binding OrderInfoCollection}">
+
+    <syncfusion:SfDataGrid.Columns >
+        <syncfusion:DataGridTextColumn HeaderText="ID" 
+                                   MappingName="OrderID" />
+        <syncfusion:DataGridTextColumn HeaderText="Customer "
+                                   MappingName="CustomerID" />
+        <syncfusion:DataGridTextColumn MappingName="Customer" />
+        <syncfusion:DataGridTextColumn HeaderText="Country"
+                                   MappingName="ShipCountry" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.AutoGenerateColumnsMode = AutoGenerateColumnsMode.None;
+
+DataGridTextColumn orderIdColumn = new DataGridTextColumn ();
+orderIdColumn.MappingName = "OrderID";
+orderIdColumn.HeaderText = "ID";
+
+DataGridTextColumn customerIdColumn = new DataGridTextColumn ();
+customerIdColumn.MappingName = "CustomerID";
+customerIdColumn.HeaderText = "Customer";
+
+DataGridTextColumn countryColumn = new DataGridTextColumn();
+countryColumn.MappingName = "ShipCountry";
+countryColumn.HeaderText = "Country";
+
+dataGrid.Columns.Add (orderIdColumn);
+dataGrid.Columns.Add (customerIdColumn);
+dataGrid.Columns.Add (countryColumn); 
+{% endhighlight %}
+{% endtabs %}
+
+## Sorting
+
+In SfDataGrid sorting can be done on its data by setting the `SfDataGrid.SortingMode` property to single,multiple or none.
+ 
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid SortingMode="Single" />
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.SortingMode=DataGridSortingMode.Single; 
+{% endhighlight %}
+{% endtabs %}
+
+Run the application and touch the header cell to sort the data and the following output will be displayed.
+ 
+
+
+Sorting can also be configured by adding the column to the `SfDataGrid.SortColumnDescription` collection as follows.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid.SortColumnDescription>
+    <syncfusion:SortColumnDescription ColumnName="CustomerID" />
+</syncfusion:SfDataGrid.SortColumnDescription> 
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.SortColumnDescription.Add (new SortColumnDescription () { ColumnName = "CustomerID" });
+{% endhighlight %}
+{% endtabs %}
+
+## Selection
+
+The SfDataGrid allows selecting the row or rows by setting the `SfDataGrid.SelectionMode` property. You can set the `SfDataGrid.SelectionMode` property to single, multiple, single deselect, or none. Information about the row or rows selected can be tracked using the `SfDataGrid.SelectedRow` and `SfDataGrid.SelectedRows` properties.
```

Comment:
The SfDataGrid allows selecting the one or more rows by setting the

Reply:
Changed the content accordingly